components/admin_console/permission_schemes_settings/permission_schemes_settings.tsx

```diff
@@ -16,27 +17,37 @@ import LoadingWrapper from 'components/widgets/loading/loading_wrapper';
 import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
 import AdminPanelWithLink from 'components/widgets/admin_console/admin_panel_with_link';
 
+import {ActionResult} from 'mattermost-redux/types/actions';
+
+import {Scheme, SchemesState} from 'mattermost-redux/types/schemes';
+
 import PermissionsSchemeSummary from './permissions_scheme_summary';
 
 const PAGE_SIZE = 30;
 const PHASE_2_MIGRATION_IMCOMPLETE_STATUS_CODE = 501;
 
-export default class PermissionSchemesSettings extends React.PureComponent {
-    static propTypes = {
-        schemes: PropTypes.object.isRequired,
-        jobsAreEnabled: PropTypes.bool,
-        clusterIsEnabled: PropTypes.bool,
-        license: PropTypes.shape({
-            CustomPermissionsSchemes: PropTypes.string,
-        }),
-        actions: PropTypes.shape({
-            loadSchemes: PropTypes.func.isRequired,
-            loadSchemeTeams: PropTypes.func.isRequired,
-        }),
-        isDisabled: PropTypes.bool,
+export type Props = {
+    schemes: SchemesState['schemes'];
+    jobsAreEnabled?: boolean;
+    clusterIsEnabled?: boolean;
+    license: {
+        CustomPermissionsSchemes: string;
+    };
+    actions: {
+        loadSchemes: (a: string, b: number, c: number) => Promise<ActionResult>;
```

Comment:
Can we have the arguments match the original method signature?
```suggestion
        loadSchemes: (scope: SchemeScope, page: number, perPage: number) => Promise<ActionResult>;
```